include/coap2/pdu.h

```diff
@@ -346,6 +346,44 @@ typedef uint8_t coap_proto_t;
 #define COAP_PROTO_TCP          3
 #define COAP_PROTO_TLS          4
 
+/**
+ * @brief Set of response codes available for a response packet.
+ *
+ * To be used when creating a response.
+ */
+enum coap_response_code {
+	COAP_RESPONSE_CODE_OK = COAP_RESPONSE_CODE(200),
+	COAP_RESPONSE_CODE_CREATED = COAP_RESPONSE_CODE(201),
```

Comment:
@Delsian Following @obgm's comment I would expect that all of the COAP_RESPONSE_ to be indented by 2 characters, not 6 characters for this enum.